package.json

```diff
@@ -22,6 +22,8 @@
   "dependencies": {
     "auth0-lock": "^10.3.0",
     "enzyme": "^2.4.1",
+    "file-saver": "^1.3.2",
+    "jquery": "^3.1.0",
     "jwt-decode": "^2.0.1",
     "karma": "^0.13.22",
```

Comment:
app.spec.js gets angry when karma is in devDependencies. that test came from auth0 though, so we might want to just get rid of it for now and handle it when we write our own tests